_data/adityarathi1998/adityarathi19.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Aditya Rathi
+institution: Shri G. S. Institute of Technology and Science, Indore
+profile_pic: adityarathi1998.jpg
```

Comment:
Your profile picture extension does not match with the file (your image is a ```.png```, not a ```.jpg```).